man/udevadm.xml

```diff
@@ -564,6 +564,18 @@
             <xi:include href="version-info.xml" xpointer="v251"/>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>-i</option></term>
```

Comment:
i don't think this should ave a short option. It's not something that should be reachable overly quickly.